dao/src/integ-test/resources/uk/gov/moj/sdt/dao/sql/PurgeDaoTest.sql

```diff
@@ -0,0 +1,90 @@
+INSERT INTO global_parameters (
+  global_parameter_id,parameter_name,parameter_value,parameter_description, version_number)
+  VALUES ( 1,'DATA_RETENTION_PERIOD',90,'Duration in days, to retain data in the tables subject to a prescribed purge', 0);
+
+INSERT INTO target_applications(
+  target_application_id, target_application_code, target_application_name, version_number)
+  VALUES (4,'MCOL','mcol live service', 1);
+
+INSERT INTO bulk_customers(
+  bulk_customer_id, sdt_customer_id, version_number, ready_for_alternate_service)
+  VALUES (1, 12345678, 1, true);
+
+INSERT INTO service_requests(
+  service_request_id, request_payload, request_timestamp, response_payload, response_timestamp, request_type, sdt_customer_id, sdt_bulk_reference, server_host_name, version_number)
+  VALUES (9, 'Req_PL09', (now() - INTERVAL '2 day'), 'Res_PL111', now(), 'OFFER', 1, 'BR109','localhost', 1);
+
+INSERT INTO service_requests(
+  service_request_id, request_payload, request_timestamp, response_payload, response_timestamp, request_type, sdt_customer_id, sdt_bulk_reference, server_host_name, version_number)
+  VALUES (3, 'Req_PL102', (now() - INTERVAL '200 day'), 'Res_PL102', now(), 'OFFER', 1, 'BR103','localhost', 1);
+
+INSERT INTO bulk_submissions(
+  bulk_submission_id, bulk_customer_id, target_application_id, service_request_id, sdt_bulk_reference, customer_reference, created_date, number_of_requests, bulk_submission_status, completed_date, updated_date, error_code, error_text, version_number, bulk_payload)
+  VALUES (11, 1, 4, 3, 'BR103', 'CR104', (now() - INTERVAL '100 day'), 1, 'BSSTATUS1', now(), now(), 'EC104', 'This is error 1', 1, 'BLK PAYLOAD 1');
+INSERT INTO bulk_submissions(
+  bulk_submission_id, bulk_customer_id, target_application_id, service_request_id, sdt_bulk_reference, customer_reference, created_date, number_of_requests, bulk_submission_status, completed_date, updated_date, error_code, error_text, version_number, bulk_payload)
+  VALUES (12, 1, 4, 3, 'BR104', 'CR105', (now() - INTERVAL '60 day'), 1, 'BSSTATUS2', now(), now(), 'EC105', 'This is error 2', 1, 'BLK PAYLOAD 2');
+INSERT INTO bulk_submissions(
+  bulk_submission_id, bulk_customer_id, target_application_id, service_request_id, sdt_bulk_reference, customer_reference, created_date, number_of_requests, bulk_submission_status, completed_date, updated_date, error_code, error_text, version_number, bulk_payload)
+  VALUES (13, 1, 4, 3, 'BR105', 'CR106', (now() - INTERVAL '30 day'), 1, 'BSSTATUS3', now(), now(), 'EC106', 'This is error 3', 1, 'BLK PAYLOAD 3');
+
+
```

Comment:
Any reason these two sets of bulk_submissions test data are separated from each other by a double blank line?  If not, remove the blank lines.